streams/src/main/java/org/apache/kafka/streams/kstream/SessionWindowedDeserializer.java

```diff
@@ -23,10 +23,20 @@
 import org.apache.kafka.streams.StreamsConfig;
 import org.apache.kafka.streams.state.internals.SessionKeySchema;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.Map;
 
 public class SessionWindowedDeserializer<T> implements Deserializer<Windowed<T>> {
 
+    /**
+     * Configuration key for the windowed inner deserializer class.
```

Comment:
Can we put any more information into this (and the other) java doc comments about the constants? Since this is a public string constant now, the use of the constant should ideally be clear from the javadoc comments.